tests/test_htmlparser.py

```diff
@@ -0,0 +1,27 @@
+import io
+
+import pytest
+
+from scripts.prepdocslib.htmlparser import LocalHTMLParser
+
+
```

Comment:
Can you add a test specifically for the cleanup function? Maybe one test for spaces, one for newlines, one for hyphens?

Reply:
Will work on it